scenarios/centos-9/edpm_ci.yml

```diff
@@ -25,7 +25,7 @@ cifmw_edpm_deploy_manifests_dir: "{{ cifmw_installyamls_repos }}/out"
 deploy_edpm: true
 
 # openshift_setup role vars
-cifmw_openshift_setup_skip_internal_registry_login: true
```

Comment:
Imho.. If we don't pull anything from the internal registry, imho.. we should skip the login.